module3/homework/inner_product/arithmeticMean.cpp

```diff
@@ -0,0 +1,14 @@
+#include <functional>
+#include <numeric>
+
+#include "arithmeticMean.hpp"
+
+double arithmeticMean(const std::vector<int>& vecA, const std::vector<int>& vecB) {
+    return std::inner_product(vecA.begin(), vecA.end(),
+                              vecB.begin(),
+                              0.0,
```

Comment:
I think that You should avoid magic numbers. What do you think about variable init{} appart from magic 0.0?

Reply:
Do you mean to create additional variable outside std::inner_product algorithm, initialise it (double val{}) and then use inside the algorithm? Or is there any trick to use such initialisation inside?